StRoot/macros/rootlogoff.C

```diff
@@ -1,10 +1,7 @@
 {
   if (TClassTable::GetDict("StMaker"))
   {
-    StMaker* mk = StMaker::GetChain();
-    if (mk) {
-      mk->Finish();
-    }
+    gROOT->Macro("rootlogoff2.C");
```

Comment:
This seems to be a more protective wrapper around `gInterpreter->ExecuteMacro()`, but I don't know that it's really any better in this circumstance. But if anyone else has a perspective on which is better to use, please chime in.

Reply:
I can't think of a simpler solution here, short of modifying our root4star application.  But that means we lose this functionality when bfc is run under root.exe.  If we can register a callback with the root ".q" command, then we would be in business.  But a very quick search didn't turn up anything.